types/fitbit-asap/app/index.d.ts

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Please rename this file to `app.d.ts` ditching `app` directory altogether, it will just work.
There is no such structure in original package